src/components/Playlist/Playlist.js

```diff
@@ -0,0 +1,38 @@
+import React, {PropTypes} from 'react';
+import LessonItem from '../LessonList/LessonItem';
+import styles from './Playlist.scss';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import {Badge, Glyphicon} from 'react-bootstrap';
```

Comment:
Som du kan lese her https://react-bootstrap.github.io/getting-started.html#commonjs så er det bedre om vi gjør
```
import Badge from 'react-bootstrap/lib/Badge';
```
istedet for
```
import {Badge} from 'react-bootstrap';
```
Vi unngår da unødvendig store filer.
Dessverre må vi da skrive importen som 2 linjer, men det overlever vi :)